usage/index.adoc

```diff
@@ -74,4 +91,26 @@ If you want to inspect the source code for all of the above it can be
 found
 https://github.com/jbosstools/jbosstools-base/tree/master/usage[here]
 
+== Error Reporting
+
+The error reporting is using https://wiki.eclipse.org/EPP/Logging[Eclipse AERI] which
```

Comment:
Error reporting uses https://wiki.eclipse.org/EPP/Logging[Eclipse AERI] is being successfuly used ...